### ISSUE TITLE
Edit content field and remove type field from winner.yml credits file

### ISSUE THREAD

**Issue by Sparky-code** [hackforla/website#2871]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, I want to edit the content field and remove the type field from a credit's yml file so that redundant code is removed and the code is easier to understand.

### Details
Currently, in each credit's yml file, there is a `content` field and a `type` field, which contains similar information. Since the fields are redundant, we decided to remove the `type` field.  

Also, we changed the `content` field to the `content-type` field to make it clearer what its purpose is. The possible values for `content-type` field are image, video, or audio; other types of content can be added if necessary. In the future, this will allow developers a way to differentiate easily between different types of content in order to show each credit's media file correctly on the website.

### Action Items
For the file `_data/internal/credits/winner.yml`, do the following:
- [ ] Change line 4 for the content field from
  ```
  content: icon
  ```
  to
  ```
  content-type: image
  ```
- [ ] Remove line 11 for the type field
  ```
  type: icon
  ```
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #2775 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #2775 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is tracked in the epic #2775.

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `_data/internal/credits/winner.yml`

- Click on the links below to see the source file code:
https://github.com/hackforla/website/blob/7164ec247825adca279d4fd99fa52895335badbb/_data/internal/credits/winner.yml#L4 
https://github.com/hackforla/website/blob/7164ec247825adca279d4fd99fa52895335badbb/_data/internal/credits/winner.yml#L11

**Comment by jond34** [comment]
i. Availability - i can start working on this this Sunday. I expect to finish this in two weeks, bythe 25th :-)

**Comment by ExperimentsInHonesty** [comment]
@jond34 When you start working on an issue please move it to the in progress column.  We have done that for you here.  Can you also please provide update
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by ExperimentsInHonesty** [comment]
Sent @jond34 a message on Slack to see if he responds there.

**Comment by ExperimentsInHonesty** [comment]
Returning this issue to the prioritized backlog

**Comment by ychang49265** [comment]
i. Availability: throughout the week/flexible
ii. ETA: one day

**Comment by ychang49265** [comment]
Blockers: 
I've added and committed my changes. However, when i attempt to pull from upstream before I push, I received the alert "You asked to pull from the remote 'upstream', but did not specify a branch. Because this is not the default configured remote for your current branch, you must specify a branch on the command line."

- Prior to working on this first issue, I created my new branch off the gh-pages branch. 
- When I enter 'git remote -v', my forked version of the repo is the origin and then hackforla/website's repo is the upstream. 
- When i tried 'git pull upstream gh-pages", I receive an alert mentioning I have divergent branches.
